sdk/python/tests/integration/feature_repos/universal/data_sources/bigquery.py

```diff
@@ -0,0 +1,50 @@
+import time
+
+import pandas as pd
+from google.cloud import bigquery
+
+from feast import BigQuerySource
+from feast.data_source import DataSource
+from tests.integration.feature_repos.universal.data_source_creator import (
+    DataSourceCreator,
+)
+
+
+class BigQueryDataSourceCreator(DataSourceCreator):
+    def teardown(self, name: str):
+        pass
+
+    def __init__(self):
+        self.client = bigquery.Client()
+
+    def create_data_source(
+        self,
+        name: str,
+        df: pd.DataFrame,
+        event_timestamp_column="ts",
+        created_timestamp_column="created_ts",
+        **kwargs,
+    ) -> DataSource:
+        gcp_project = self.client.project
+        bigquery_dataset = "test_ingestion"
+        dataset = bigquery.Dataset(f"{gcp_project}.{bigquery_dataset}")
+        self.client.create_dataset(dataset, exists_ok=True)
+        dataset.default_table_expiration_ms = (
```

Comment:
Would we need expiration if we are explicitly doing cleanup?

Reply:
We wouldn't